test/JDBC/expected/BABEL-3592.out

```diff
@@ -77,27 +88,25 @@ GO
 text
 Query Text: EXEC babel_3592_insert_multiline
   Query Text: insert into babel_3592_t2 select * from babel_3592_t1 where b1 = 1
-  ->  Insert on babel_3592_t2
-        ->  Bitmap Heap Scan on babel_3592_t1
-              Recheck Cond: (b1 = 1)
-              ->  Bitmap Index Scan on index_babel_3592_t1_b1babel_35976c64b612d1f74e2768783beca3bf836
-                    Index Cond: (b1 = 1)
+  ->  Insert on babel_3592_t2  (cost=0.00..0.00 rows=0 width=0)
```

Comment:
Why there's still cost in here ?

Reply:
This is experiment PR to understand difference